msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -10,11 +10,13 @@ ms.date: 04/07/2022
 
 # Apps for Teams meetings and calls
 
-Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting app can deliver a user experience for each stage of the meeting lifecycle. Meeting lifecycle includes pre-meeting, in-meeting, and post-meeting app experience, depending on the attendee's status.
+Meetings enable collaboration, partnership, informed communication, and shared feedback. The meeting space can deliver a user experience for each stage of the meeting lifecycle. The following image gives you an idea of the meeting app extensibility:
 
-> [!Note]
->
-> Apps for instant meetings, scheduled public channel meetings, one-on-one, and group calls are currently available only in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+:::image type="content" source="../assets/images/apps-in-meetings/meetingappextensibility.png" alt-text="The screenshot shows you how meeting app extensibility works.":::
+
+An application developer must be familiar with the following product concepts to create custom meeting experiences in Microsoft Teams with apps.
+
```

Comment:
After this intro, I would recommend a short summary of the different capabilities of meeting apps (tabs, stage, bot, etc.). This page is a chance to really sell our meeting features and make them seem exciting!

Reply:
differing for next iteration https://dev.azure.com/domoreexp/MSTeams/_workitems/edit/2694775